app/assets/javascripts/game.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+  function move_piece(url,new_x_position,new_y_position) {
```

Comment:
It's a real mind-eff, but Javascript naming convention for methods is lowerCamelCase. For the most part, anything that you would normally use snake_case_for_in_rails you would useLowerCamelCaseInJavaScript